tests/run.sh

```diff
@@ -12,5 +12,15 @@ function finish {
 }
 trap finish EXIT
 
-sleep 5
-python -u testrunner/bin/trunner -s http://localhost:5000
+# run docker and remove container afterwards
+#   where our SDK server is located
+#   host-relative path to fixtures mountpoint
+#   configure the fixtures mountpoint
+#   configure the results mountpoint
+#   image name
+
+docker run --rm --net=host --name=testrunner_container \
+-e "TEST_SERVER_URL=http://127.0.0.1:5000" \
```

Comment:
nit: these env vars could have defaults encapsulated in your docker image.

Reply:
We do actually have it defaulted to `http://localhost:5000` but I think it's best to be explicit here, just in case the default changes for any reason. There's not much config needed for the testrunner, but the server is an important one.